### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by nt4f04uNd**
Fixes https://github.com/nt4f04uNd/sweyer/issues/155

Here are other settings in GitHub UI 
image

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+updates:
+  - package-ecosystem: "pub"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "pub"
+    directory: "/sweyer_plugin"
+    schedule:
+      interval: "daily"
```
Comment (Abestanis) [inline .github/dependabot.yml:16]:
```suggestion
      interval: "daily"

 - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"

```
Let's also let it keep the actions in our CI up to date.

**Comment by Abestanis** [comment]
Don't we want to enable the the `Dependabot version upgrades`? The point was to get some automatic PRs when there are new versions.

**Comment by nt4f04uNd** [comment]
> Don't we want to enable the the `Dependabot version upgrades`? The point was to get some automatic PRs when there are new versions.

This PR is what it takes to enable it :-)

**Comment by Abestanis** [comment]
> This PR is what it takes to enable it :-)

Ah, I didn't realize that. Perfect.

File: .github/dependabot.yml (hunk 1 above)
Comment (Abestanis) [reply .github/dependabot.yml:16]: [Docs](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot#example-dependabotyml-file-for-github-actions)

**Review by Abestanis — APPROVED** [review]

**Pull request merged**